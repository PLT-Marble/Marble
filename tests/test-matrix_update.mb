main() {
    matrix a = [1.1,2.2;3.3,4.4];
    a[0,1] = 5.5;
    printmf(a);
    return 0;
}