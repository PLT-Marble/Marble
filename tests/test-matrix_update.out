1.100000 5.500000
3.300000 4.400000